apps/summon-app/src/app/constants.ts

```diff
@@ -0,0 +1,27 @@
+// FOR DEMONSTRATIOn
```

Comment:
Makes sense to include this! We can add to some docs (I'll add to the list) explaining this and the env vars that are needed (and how to set them up)